styles.css

```diff
@@ -0,0 +1,125 @@
+body,
+h1,
+h2 {
+  margin: 0;
+}
+
+nav {
+  width: 100%;
+  height: 72px;
+  background: #3e3c3c;
+  display: flex;
+  align-items: center;
+  padding: 26px 0;
+  box-sizing: border-box;
+  justify-content: space-between;
+}
+
+.navbar-logo {
+  font-size: 20px;
+  line-height: 20px;
+  color: #fff;
+}
```

Comment:
Please ensure that you stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
 
![your_design](https://user-images.githubusercontent.com/10332499/153313610-6cb87264-90f7-4afc-8175-58b11def1c40.PNG) | ![figma_design](https://user-images.githubusercontent.com/10332499/153313627-5f78b5d0-8635-4407-8f9c-fcb776c67e8c.PNG)

Reply:
The page is now responsive, and the sizing is now the same as Figma design, but the font which is used in Figma is called "Crete Round" and I used the same font from google, but it looks a bit different.
I checked and the font is applied on text but it looks a bit different, and I don't know why?

Reply:
Oh I forget to mention you @micheaol 😊